data/helpers.d/setting

```diff
@@ -259,23 +261,30 @@ ynh_webpath_register () {
 #
 # Requires YunoHost version 3.7.0 or higher.
 ynh_permission_create() {
-    declare -Ar args_array=( [p]=permission= [u]=url= [a]=allowed= )
+    declare -Ar args_array=( [p]=permission= [u]=url= [a]=allowed= [p]=is_protected= )
     local permission
     local url
     local allowed
+    local is_protected
     ynh_handle_getopts_args "$@"
 
     if [[ -n ${url:-} ]]; then
         url="'$url'"
     else
         url="None"
     fi
-
     if [[ -n ${allowed:-} ]]; then
         allowed=",allowed=['${allowed//';'/"','"}']"
     fi
+    if [ -n ${is_protected} ]; then
+        if [ $is_protected == "true" ]; then
+            is_protected=",is_protected=$is_protected=True"
```

Comment:
I didn't tested, but

```suggestion
            is_protected=",is_protected=True"
```
That sounds more like what you want to do here? (Or I'm missing something)

Same for `false`, and in `ynh_permission_update()`

Reply:
Yes :+1: